Tests/YamsTests/MarkTests.swift

```diff
@@ -39,6 +39,24 @@ class MarkTests: XCTestCase {
                 "'identifier_name' and will be completely removed in a future release."
             ])
     }
+
+    func testMappingMarkIsCorrect() throws {
```

Comment:
Yams manually tracks tests for Linux in `allTests`, so could you please add this test in the `allTests` extension below?